java/ql/lib/ext/jakarta.ws.rs.core.model.yml

```diff
@@ -3,6 +3,7 @@ extensions:
       pack: codeql/java-all
       extensible: sinkModel
     data:
+      - ["jakarta.ws.rs.core", "Response", False, "created", "(URI)", "", "Argument[0]", "request-forgery", "ai-manual"] # it's unclear whether this does some sanitization (in which case it's likely a negative)
```

Comment:
In any case, this is building an [`HTTP 201 Created` response](https://www.rfc-editor.org/rfc/rfc9110#status.201) and setting the `URI` parameter as the `Location` header, but not making a request to it. So I don't think `request-forgery` fits.

```suggestion
```